### PR TITLE
Drivers implementation

### PR DESCRIPTION
**Pull request by Abyss777**
Web part for https://github.com/tananaev/traccar/pull/3349

image

Format Driver in such way: "driverUniqueId (driverName)" if name is present, otherwise uniqueId only.

### CODE REVIEW

File: web/l10n/en.json [hunk 1]
```diff
@@ -269,6 +272,7 @@
     "eventIgnitionOff": "Ignition is OFF",
     "eventMaintenance": "Maintenance required",
     "eventTextMessage": "Text message received",
+    "eventDriverUnauthorized": "Driver Unauthorized",
```
Comment (tananaev) [inline web/l10n/en.json]: Do we need this event? I think instead we should have an event when a new driver has authorised. I believe in most cases this feature will be used for informational purposes only, not to detect unauthorised access.

Comment (Abyss777) [inline web/l10n/en.json]: May be you are right. It is very unlikely when somebody that has rfid mark take not his car and put the wrong mark on the reader.

Comment (tananaev) [inline web/l10n/en.json]: Exactly, but it would be nice to have event for driver change.

File: web/l10n/en.json [hunk 2]
```diff
@@ -269,6 +272,7 @@
     "eventIgnitionOff": "Ignition is OFF",
     "eventMaintenance": "Maintenance required",
     "eventTextMessage": "Text message received",
+    "eventDriverChanged": "Driver Changed",
```
Comment (tananaev) [inline web/l10n/en.json]: I think text should be consistent with other events, which means only word should be capitalised.

**Comment by Abyss777** [comment]
Replaced commit - make string consistence with other events

**Comment by tananaev** [comment]
Merged, thanks.

**Pull request merged**